### PR TITLE
fix(docs): enable flex search

### PR DESCRIPTION
**Pull request by tknickman**
### Description

Fix docs search

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -27,10 +28,11 @@
     "framer-motion": "^7.3.6",
     "next": "^13.3.4",
     "next-themes": "^0.2.0",
-    "nextra": "^2.4.2",
-    "nextra-theme-docs": "^2.4.2",
+    "nextra": "^2.1.0",
+    "nextra-theme-docs": "^2.1.0",
```
Comment (tknickman) [inline docs/package.json:32]: This and the i18n change is what actually helped here

File: docs/package.json [hunk 2]
```diff
@@ -27,10 +28,11 @@
...
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "sharp": "^0.32.1",
```
Comment (tknickman) [inline docs/package.json:35]: recommended for next/image

File: docs/next.config.js [hunk 3]
```diff
@@ -2,6 +2,9 @@ const { withSentryConfig } = require("@sentry/nextjs");
 const withNextra = require("nextra")({
   theme: "nextra-theme-docs",
   themeConfig: "./theme.config.tsx",
+  // options
+  flexsearch: true,
```
Comment (tknickman) [inline docs/next.config.js:6]: This is the default, but I left it in here for verbosity

**Review by arlyon — APPROVED** [review]

**Comment by arlyon** [comment]
Looks like the preview fixed it :)

**Pull request merged**